_episodes/05-mplhep.md

```diff
@@ -269,6 +269,12 @@ fig.show()
 {: .challenge}
 
 > ## Solution
+> Theory:
+> First, rest mass is not a conserved quantity.
+> The important idea is that in Higgs rest frame two Z bosons of 90 GeV each cannot be produced.
```

Comment:
I should have been more clear. I meant two "on-shell" Z bosons cannot be produced.

 I will add "on-shell"

Reply:
But I think this then basically reduces the explanation to new possible-not-known-words "on-shell" and "off-shell", right?  And that's not really providing more answers, just new names. 

So I like what you alluded below with the energy being just an expectation value etc., because that's an actual statement :)  Perhaps we can expand on that?

Reply:
for on-shell and off-shell I have included a statement on what that meants: 

 (On-shell particles are the ones that obey $$ E^2 = p^2 + m^2 $$ whereas off-shell particles do not.)

we could also include the statement that energy is just an expectation value. for instance the same as:

Energy of Z boson (or any decaying particle) is the expectation value of energy which is inherently probabilistic. Energy and momentum are always conserved.

Reply:
shall I include that?